SU-HSDO/suhumsci
# STN-343 : Implement the Traditional hero and carousel styles

# [STN-343](https://sparkbox.atlassian.net/secure/RapidBoard.jspa?rapidView=80&modal=detail&selectedIssue=STN-343) READY FOR REVIEW

## Summary
Create the Hero and Carousel for the Traditional theme. They share the same functionality, behaviors, and use-cases as the colorful theme hero and carousel but with different styling.

## Need Review By (Date)
asap

## Urgency
high

## Steps to Test
1. In the CLI, run `npm run test` and make sure all of our Sass tests pass.
2. Review the styles for Traditional hero and carousel components when they are placed within the top of the page and when they are placed with the main content of the page. Make sure they match the design comps: https://www.figma.com/file/bG42fKKioEcnJ0ORKkLL5E/⚙%EF%B8%8F-Traditional-Theme---Working?node-id=5686%3A2
3. The QA Page http://economics.suhumsci.loc/qa/qa-page has a couple implementations of the hero and carousel. I recommend creating an additional page to test various scenarios such as:
* An image, heading, copy, and link
* An image, heading, and copy
* An image and heading
* No image with a heading, copy, and/or link 
4. Review the Colorful hero and carousel and verify nothing has changed.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
